build.gradle

```diff
@@ -27,6 +28,8 @@ allprojects {
     repositories {
         jcenter()
         maven { url 'https://maven.google.com' }
+        google()
```

Comment:
`maven { url 'https://maven.google.com' }` is equals to `google()`
I prefer to keep just `google()`. Please remove maven { url 'https://maven.google.com' }

https://stackoverflow.com/questions/46467561/difference-between-google-and-maven-url-https-maven-google-com

Reply:
Just pushed an update. I prefer `google()` also. Bypassed removing the longer statement.